custom_components/tuya_local/devices/kcvents_vt501_fan.yaml

```diff
@@ -0,0 +1,153 @@
+name: Heat Recovery Ventilator
+products:
+  - id: 5p0togo3y3yzgp2p
+    name: KCvents VT-501
+primary_entity:
+  entity: fan
+  translation_only_key: fan_with_presets
```

Comment:
Since none of the presets match anything in this translation key, I think it would be better to use `ventilation`, which has presets `fresh`, `circulate`, and maybe `anti-condensation`, to use in place of "Fresh Air Mode", "Regenerate Mode" and "Exhausted Air Mode" below. The Low, Medium and High humidity presets could maybe be added to this key if they are mutually exclusive options from the other three. But if these can be set independently, it would be better to break them out into another setting.

Reply:
I will change it, make sense. Thank you.
Yes, the modes are mutually exclusive, so there are these 6 modes in total.